src/Vim.xml

```diff
@@ -0,0 +1,120 @@
+<SPDX name="Vim License" identifier="Vim" osi-approved="false">
```

Comment:
Original license text contains texted bullets, where as the bullets here have been flattened to the top-level. Does the XML format permit nesting bullets, and if so, should we use that here?